tools/update-rules-list.ts

```diff
@@ -30,9 +30,9 @@ const rulesList: TRulesList[] = Object.entries(rules)
       createRuleLink(rule.name),
       rule.meta.docs.description,
       rule.meta.fixable ? emojiKey.fixable : '',
-      rule.meta.docs.categories
+      rule.meta.docs.categories && rule.meta.docs.categories.length > 0
         ? `<ul>${rule.meta.docs.categories.map((c) => `<li>${c}</li>`).join('')}</ul>`
-        : '',
+        : '(none)',
```

Comment:
This part is no longer needed now as there is a mechanism to ignore rules without categories, and I've been able to remove it entirely on my rebased branch.